internal/commands/login/inputs_test.go

```diff
@@ -184,10 +205,38 @@ func TestLoginInputs(t *testing.T) {
 
 			assert.Nil(t, tc.inputs.Resolve(profile, ui))
 
+			assert.Equal(t, tc.expectedURL, expectedURL)
```

Comment:
[question] I think it matters not, but maybe we want to move this assert to after the `<-doneCh` receive?  that's technically the point when "inputs are done resolving" (even if the browser portion of input resolution is synchronous currently)